setup.py

```diff
@@ -8,7 +8,7 @@
     version='0.1.1',
     description='Reusable automation and configuration utilities for edX Django applications',
     author='edX',
-    url='https://github.com/edx/edx-django-extensions',
+    url='https://github.com/edx/edx-management-commands',
```

Comment:
Please bump the patch version and tag a new release when this lands so that this change is reflected on PyPI.